### ISSUE TITLE
MultiUploader.setEnabled(false) doesn't disable the "delete" button 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bdshadow/gwtupload#204]
```
after calling setEnabled(false) on my MultiUploader instance, the "Browse" 
button is disabled.

But I would also like to have the red-"X" icon which deletes the file disabled, 
and this sadly is not happening.

Could you fix this or tell me how to work around it in GWT code? Is there a 
public access path to this icon and what is it?
```

Original issue reported on code.google.com by `kaef...@gmail.com` on 7 Nov 2014 at 4:28

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 16eac6771a2b.
```

Original comment by `manuel.carrasco.m` on 13 Jan 2015 at 10:38
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `manuel.carrasco.m` on 13 Jan 2015 at 5:41
- Added labels: **Milestone-1.0.2**
